benchmarks/benchmarks.js

```diff
@@ -0,0 +1,4 @@
+var drain = require("./drain");
+var benchmark = require("./scaffold");
+
+benchmark.run();
```

Comment:
If we call this file `index.js`, we can run `node benchmarks` to execute them all.